contracts/TransitionApplier2.sol

```diff
@@ -138,6 +133,21 @@ contract TransitionApplier2 {
         return (_accountInfo, _globalInfo);
     }
 
+    /**
+     * @notice Apply a UpdateEpochTransition.
+     *
+     * @param _transition The disputed transition.
+     * @param _globalInfo The involved global info from the previous transition.
+     * @return new global info after applying the disputed transition
+     */
+    function applyUpdateEpochTransition(
+        dt.UpdateEpochTransition memory _transition,
+        dt.GlobalInfo memory _globalInfo
+    ) public pure returns (dt.GlobalInfo memory) {
+        _globalInfo.currEpoch = _transition.epoch;
```

Comment:
For extra safety, only override if `_transition.epoch >= _globalInfo.currEpoch`?

Reply:
updated, though I don't think it is necessary, because `checkPendingEpochUpdate` enforce the order